multipart/multipart.py

```diff
@@ -304,7 +304,7 @@ def __eq__(self, other: object) -> bool:
             return NotImplemented
 
     def __repr__(self) -> str:
-        if len(self.value) > 97:
+        if self.value is not None and len(self.value) > 97:
```

Comment:
How `self.value` can be `None`?

Reply:
If set by `set_none`.

It is also type hinted here:
```python
    @property
    def value(self) -> bytes | None:
```